server.js

```diff
@@ -4,6 +4,15 @@ var bodyParser = require('body-parser');
 var app = express();
 app.use(bodyParser.json());
 
+function handlePR_Event(reqBody) {
+    //there are three types of events.
+    //1. pull request.
+    //2. pull request review comment.
+    //3. pull request review.
```

Comment:
This seems related to https://github.com/luisschubert/webhookTest/issues/3

(this is false, it's not related, just linking for fun and profit).